index.js

```diff
@@ -17,7 +17,11 @@ module.exports = input => {
 		}, opts);
 
 		for (let i = 0; i < header.length; i++) {
-			if (header[i] !== buf[i + opts.offset]) {
+			if (opts.mask) {
```

Comment:
Can you add a comment about what the `mask` option does? For posterity.

Reply:
comment in c433cf3b6b23691709a3f7c4fa2114a74b190ccb